sdks/python/apache_beam/transforms/periodicsequence.py

```diff
@@ -216,6 +217,21 @@ def expand(self, pcoll):
         | 'MapToTimestamped' >> beam.Map(lambda tt: TimestampedValue(tt, tt)))
 
 
+class RebaseMode(enum.Enum):
```

Comment:
Java/Python PeriodicImpulse API now gradually deviate. In Java side there is a "stopAfter"option that make start timestamp gets stamped at the beginning of pipeline execution